_includes/manuals/1.11/2.1_quickstart_installation.md

```diff
@@ -147,7 +147,8 @@ yum -y install http://yum.theforeman.org/releases/{{page.version}}/f21/x86_64/fo
 {% highlight bash %}
 echo "deb http://deb.theforeman.org/ jessie {{page.version}}" > /etc/apt/sources.list.d/foreman.list
 echo "deb http://deb.theforeman.org/ plugins {{page.version}}" >> /etc/apt/sources.list.d/foreman.list
-wget -q http://deb.theforeman.org/pubkey.gpg -O- | apt-key add -
+apt-get -y install ca-certificates
+wget -q https://deb.theforeman.org/pubkey.gpg -O- | apt-key add -
```

Comment:
Ah good point, I missed that. Let me rephrase then - do we want to recommend an https repo? I'm in two minds....